.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
+    "cSpell.words": [
+        "argmemonly",
+        "getelementptr",
+        "inlinehint",
```

Comment:
This is a nit, but for the spec document is it appropriate to have the .vscode settings in the repo? shouldn't these be set somewhere globally within your own environment?

Reply:
Well, VS Code does offer nice things that are also enforced by our CI (like spell checking and formatting for markdown). I hence would say it is appropriate given that CI won't pass unless you format the files appropriately. I actually also meant to add the necessary extensions to do spell checking etc. to the recommended extensions.

Reply:
Sounds reasonable!